Source/PawnRenderer_Zombiefied.cs

```diff
@@ -431,6 +406,8 @@ private void DrawDebug()
         {
             if (DebugViewSettings.drawDuties && Find.Selector.IsSelected(this.pawn) && this.pawn.mindState != null && this.pawn.mindState.duty != null)
             {
+                // Well, we should probably do some kind of DrawDebug at some point, right?
```

Comment:
Maybe yea but this file is outdated anyways. It was copied from decompiled Rimworld code and then adjusted to allow rendering Zombies as humanlikes even when theyre animals under the hood.

The pawn rendering changed a lot in Rimworld so now there are some problems like Zombies wounds are not displayed. This file needs a redo

Reply:
Relevant file is PawnRenderer if you want to take a look. Right now zombies dont show any wounds that annoys me

Reply:
For sure something I'm curious about. I feel like I'd get a lot of use out of a debug drawing method with some of situations I've stumbled upon so far, though maybe that's cause I'm not too adept at debugging this stuff yet